cmd/build_test_matrix/main.go

```diff
@@ -43,10 +44,20 @@ func main() {
 	fmt.Println(string(ghBytes))
 }
 
+// getTestSuiteToRun returns the specified test suite name to run if present, otherwise
+// it returns an empty string which will result in running all test suites.
+func getTestSuiteToRun() string {
+	testSuite, ok := os.LookupEnv(testSuiteEnv)
+	if !ok {
+		return ""
+	}
+	return testSuite
+}
+
 // getGithubActionMatrixForTests returns a json string representing the contents that should go in the matrix
 // field in a github action workflow. This string can be used with `fromJSON(str)` to dynamically build
 // the workflow matrix to include all E2E tests under the e2eRootDirectory directory.
-func getGithubActionMatrixForTests(e2eRootDirectory string) (GithubActionTestMatrix, error) {
+func getGithubActionMatrixForTests(e2eRootDirectory, suite string) (GithubActionTestMatrix, error) {
```

Comment:
Should we add godoc to specify the usage of an empty string to indicate runAllTests?